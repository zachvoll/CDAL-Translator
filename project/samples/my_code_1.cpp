#include <stdio.h>
#include <stdlib.h>
#include <iostream>
#include <math.h>
#include "Matrix.h"
using namespace std;

int main() {
 matrix a = matrix::matrixRead("../samples/my_code_data1.data");
 ;  matrix m = matrix::matrixRead("../samples/my_code_data2.data");
 ;  matrix result = a * m;
 ; cout << result;
 
return 0;
}
