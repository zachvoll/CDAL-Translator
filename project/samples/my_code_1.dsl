/* This program is just a copy of sample_1.dsl. You should write your CDAL program to multiply two
2-dimensional matrices and save it
in a file with this name. */

main () {

    matrix a = matrixRead ( "../samples/my_code_data1.data" ) ;

    matrix m = matrixRead ( "../samples/my_code_data2.data" ) ;

    matrix result = a * m ; 

    print ( result ) ; 

}
