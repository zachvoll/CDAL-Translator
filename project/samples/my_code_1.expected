2 2
40  40  
40  40  
